### DISCUSSION TITLE
Report bugs?

### DISCUSSION THREAD

**Discussion by willeke1234** [GrandOrgue/grandorgue#1122] [Developers]
I was wondering, are issues like below considered bugs? Are such issues going to be fixed? Should I report such issues and where? Should I report what I consider bad GUI?

Names of temperaments do not get translated #1104
Settings, Reverb, warning that can savely be ignored #1050
Size of the organ load selection window #1051
Error log on startup #1109

**Comment by kerkovits** [comment]
https://github.com/GrandOrgue/grandorgue/discussions/1109 was fixed by https://github.com/GrandOrgue/grandorgue/pull/1106
https://github.com/GrandOrgue/grandorgue/issues/1104 might get fixed by https://github.com/GrandOrgue/grandorgue/pull/1117

**Reply by willeke1234** [reply]
#1106 fixed #1093 and introduced #1109

**Comment by willeke1234** [comment]
I still have these questions:

I was wondering, are issues like below considered bugs? Are such issues going to be fixed? Should I report such issues and where? Should I report what I consider bad GUI?

Settings, Reverb, warning that can savely be ignored #1050
Size of the organ load selection window #1051
Error log on startup #1109

**Comment by oleg68** [comment]
Generally, lack of a desired feature can not be considered as a bug.

#1050 It reflects the current state of the reverb: lack of it's oficial support. I don't know all caveats of this functionality.
#1051: lack of a feature: now only the initial toolbar window and UI panels (from the Panel menu) save their positions. Any dialogs (including the organ selection dialog) do not save their positions. You may file a feature request for it.
#1109: it was a bug in some previous version that wrote incorrect entries to the preset file. This bug was fixed a long time ago, so the warning when opening a preset saved with the old buggy GO version is **a normal behavior for the current GO version**.

**Comment by willeke1234** [comment]
I'll report anything I think needs improvement, including GUI issues.
